examples/server/pkg/lifecycle_v1alpha2/lifecycle_v1alpha2.go

```diff
@@ -0,0 +1,114 @@
+// Copyright 2019 VMware, Inc. All Rights Reserved.
```

Comment:
So far we have decided and is in the spec that the unique ID of the FederatedService is the Name. From the spec: A FederatedService is uniquely idenfied in the owner platform by its name.

Reply:
the same comment as previous, we are not looking at name or fqdn in this MR, it's primarily using resource-id.